portfolio/src/main/webapp/js/script.js

```diff
@@ -53,9 +53,39 @@ async function getText() {
   }
 }
 
+/**
+ * Display text from data Servlet
+ */
+async function getText() {
+  var numComments = document.getElementById("number-comments");
+  numComments = numComments.options[numComments.selectedIndex].value;
+  getNumComments(numComments);
+}
+
 /** Creates an <li> element containing text. */
-function createListElement(text) {
+function createListElement(message) {
```

Comment:
looks like you're changing the argument type. Now might be a good time to familiarize yourself with proper documentation of arguments that look like: `@param {string} someString`.

Please see https://google.github.io/styleguide/jsguide.html#features-functions-parameter-return-types for more details on this.